BGL/include/CGAL/boost/graph/boost_parameters_interface.h

```diff
@@ -19,6 +19,6 @@
 
 
 
-CGAL_add_pmp_parameter(vertex_index_t, vertex_index_map, vertex_index_map)
-CGAL_add_pmp_parameter(edge_index_t, edge_index_map, edge_index_map)
-CGAL_add_pmp_parameter(graph_visitor_t, visitor, visitor)
+CGAL_add_named_parameter(vertex_index_t, vertex_index_map, vertex_index_map)
```

Comment:
Actually the guidelines say : 
- Names of constants are uppercase (e.g., ORIGIN).
- Names of macros should begin with the prefix CGAL_.
 So I think it does match the naming convention. Furthermore, macros in interface_macros.h are named CGAL_Kernel_obj, etc. 
 I agree that I should make the A of Add uppercase, but following the guidelines does not lead to CGAL_ADD_PMP_PARAMETER.